static/js/util.js

```diff
@@ -115,6 +115,18 @@ function arrSum(arr) {
 }
 
 
+/**
+ * Calculate the average of all elements in the input array.
+ * @fcuntion arrMean
+ * @param {number[]} arr - input array
+ * returns {number} mean
+ */
+function arrMean(arr) {
+  // to avoid floating point err in js
+  return (arrSum(arr) * 10) / (arr.length * 10);
```

Comment:
This is interesting! Can you point me to a reference / add reference to docstring about the JS floating point error?

Reply:
https://stackoverflow.com/questions/1458633/how-to-deal-with-floating-point-number-precision-in-javascript
https://www.w3schools.com/js/js_numbers.asp

for example, 3.3/3 returns 1.0999999999999999 instead of 1.1